packages/itwinui-react/src/core/Table/cells/DefaultCell.tsx

```diff
@@ -47,12 +50,15 @@ export const DefaultCell = <T extends Record<string, unknown>>(
   props: DefaultCellProps<T>,
 ) => {
   const instance = React.useContext(TableInstanceContext);
+
   const isCustomCell = React.useMemo(
     () =>
       instance?.columns.find(({ id }) => props.cellProps.column.id === id)
         ?.Cell !== defaultColumn.Cell,
     [instance, props.cellProps.column.id],
   );
+  const isCustomRenderer =
```

Comment:
This is no longer checking for `cellRenderer` and so should be named something like `hasCustomChildren` instead?